src/context/WalletProvider/NativeWallet/components/NativeCreate.tsx

```diff
@@ -15,14 +16,15 @@ import { GENERATE_MNEMONIC, Vault } from '@shapeshiftoss/hdwallet-native-vault'
 import { range } from 'lodash'
 import { Component, useEffect, useMemo, useRef, useState } from 'react'
 import { FaEye } from 'react-icons/fa'
+import { useTranslate } from 'react-polyglot'
 import { Text } from 'components/Text'
 
 import { NativeSetupProps } from '../types'
 
-const getVault = async (): Promise<Vault> => {
+const getVault = async (mnemonic?: string): Promise<Vault> => {
```

Comment:
We won't pass a mnemonic into here. Ideally, we'll pass a vault into this component, and if a vault already exists, we will skip the getVault function.